tests/unit/test_semver.py

```diff
@@ -145,3 +146,20 @@ def test_largerthan_equal():
         semver.create_semver_matcher(">=1.2.3").match(semver.str_to_version("1.2.2"))
         == False
     )
+
+
+@pytest.mark.parametrize(
+    "vstr, expected",
+    [
```

Comment:
Do we need a test case for suffixes (`1.2.3-rc4`)?

Reply:
Don't think so, since we only use this for exact or truncated versions.